KarhooUISDK/Screens/BookingRequestScreen/FormBookingRequestScreen/FormBookingRequestViewController.swift

```diff
@@ -118,129 +225,154 @@ final class FormBookingRequestViewController: UIViewController, BookingRequestVi
+             mainStackContainer.trailingAnchor.constraint(equalTo: container.trailingAnchor)]
+            .map { $0.isActive = true }
+        
+        mainStackBottomPadding = mainStackContainer.bottomAnchor.constraint(equalTo: container.bottomAnchor,
+                                                                            constant: -20.0)
+        mainStackBottomPadding.isActive = true
+        
+        _ = [termsConditionsView.leadingAnchor.constraint(equalTo: mainStackContainer.leadingAnchor),
+             termsConditionsView.trailingAnchor.constraint(equalTo: mainStackContainer.trailingAnchor)]
+            .map { $0.isActive = true }
+        
+        _ = [separatorLine.heightAnchor.constraint(equalToConstant: 1.0),
+             separatorLine.leadingAnchor.constraint(equalTo: mainStackContainer.leadingAnchor),
+             separatorLine.trailingAnchor.constraint(equalTo: mainStackContainer.trailingAnchor)]
+            .map { $0.isActive = true }
+    }
+    
+    private func setupConstraintsForDefault() {
+        _ = [exitButton.topAnchor.constraint(equalTo: container.topAnchor, constant: view.safeAreaInsets.top),
+             exitButton.trailingAnchor.constraint(equalTo: container.trailingAnchor),
+             exitButton.widthAnchor.constraint(equalToConstant: 50.0)].map { $0.isActive = true }
+        
+        container.heightAnchor.constraint(equalToConstant: UIScreen.main.bounds.height).isActive = true
+        
+        containerBottomConstraint = container.bottomAnchor.constraint(equalTo: view.bottomAnchor,
+                                                                      constant: UIScreen.main.bounds.height)
+        containerBottomConstraint.isActive = true
+        _ = [baseStackView.topAnchor.constraint(equalTo: exitButton.bottomAnchor),
+             baseStackView.leadingAnchor.constraint(equalTo: container.leadingAnchor),
+             baseStackView.trailingAnchor.constraint(equalTo: container.trailingAnchor),
+             baseStackView.bottomAnchor.constraint(equalTo: container.bottomAnchor)].map { $0.isActive = true }
+        
+        let titleInset: CGFloat = 15.0
+        
+        _ = [headerView.leadingAnchor.constraint(equalTo: baseStackView.leadingAnchor, constant: titleInset),
+             headerView.trailingAnchor.constraint(equalTo: baseStackView.trailingAnchor,
+                                                  constant: -titleInset)].map { $0.isActive = true }
+        
+        _ = [passengerDetailsTitle.leadingAnchor.constraint(equalTo: baseStackView.leadingAnchor,
+                                                            constant: titleInset),
+             passengerDetailsTitle.trailingAnchor.constraint(equalTo: baseStackView.trailingAnchor,
+                                                             constant: -titleInset)].map { $0.isActive = true }
+        
+        passengerDetailsView.pinLeftRightEdegs(to: baseStackView)
```

Comment:
perhaps we can use the below (we are hiding it for KarhooUsers so maybe we dont need it in that case

```
        if presenter.isKarhooUser() == false {
            passengerDetailsView.pinLeftRightEdegs(to: baseStackView)
        }
```

Reply:
I use these constraints by default, hiding/displaying views if needed